src/main/java/com/contentful/java/cda/image/ImageOption.java

```diff
@@ -0,0 +1,334 @@
+    return new ImageOption("f", focus.name());
+  }
+
+  /**
+   * Cut corners round.
+   *
+   * @param radius of the inner circle in pixel
+   * @return an image option for manipulating a given url.
+   * @throws IllegalArgumentException if the radius is negative.
+   */
+  public static ImageOption roundedCornerRadiusOf(float radius) {
+    if (radius < 0) {
+      throw new IllegalArgumentException("Radius is negative.");
+    }
+
+    return new ImageOption("r", Float.toString(radius));
+  }
+
+  /**
+   * Define a background color.
+   * <p>
+   * The color value must be a hexadecimal value, i.e. 0xFF0000 means red.
+   *
+   * @param color the color in hex to be used.
+   * @return an image option for manipulating a given url.
+   * @throws IllegalArgumentException if the color is less then zero or greater then 0xFFFFFF.
+   */
+  public static ImageOption backgroundColorOf(int color) {
+    if (color < 0 || color > 0xFFFFFF) {
+      throw new IllegalArgumentException("Color must be in rgb hex range of 0x0 to 0xFFFFFF.");
+    }
+    return new ImageOption("bg", "rgb:" + format(Locale.getDefault(), "%06X", color));
+  }
+
+  /**
+   * Define a background color by its components.
+   * <p>
+   * Each color component must be a hexadecimal in the range from 0 to 255 inclusively.
+   *
+   * @param r the red color component in hex to be used.
+   * @param g the green color component in hex to be used.
+   * @param b the blue color component in hex to be used.
+   * @return an image option for manipulating a given url.
+   * @throws IllegalArgumentException if a component is less then zero or greater then 255.
+   */
+  public static ImageOption backgroundColorOf(int r, int g, int b) {
+    if (r > 255 || r < 0) {
+      throw new IllegalArgumentException("Red component out of range: " + r);
+    }
+    if (g > 255 || g < 0) {
+      throw new IllegalArgumentException("Green component out of range: " + g);
+    }
+    if (b > 255 || b < 0) {
+      throw new IllegalArgumentException("Blue component out of range: " + b);
+    }
+
+    return new ImageOption("bg", "rgb:" + format(Locale.getDefault(), "%02X%02X%02X", r, g, b));
+  }
+
+  /**
+   * Define a background color as black.
+   *
+   * @return an image option for manipulating a given url.
+   */
+  public static ImageOption blackBackgroundColor() {
+    return backgroundColorOf(0);
+  }
+
+  /**
+   * Prefix image urls with http.
+   * <p>
+   * If a protocol (either http or https) was already specified, this one will be taken and
+   * this option will not have any effect.
+   *
+   * @return an image option changing a given url to use insecure http transport.
+   */
+  public static ImageOption http() {
+    return new ImageOption("http", "") {
+      @Override public String apply(String url) {
+        if (url.startsWith("//")) {
+          return "http:" + url;
+        } else {
+          return url;
+        }
+      }
+    };
+  }
+
```

Comment:
Some servers have a hard time using https, so to have it aligned with the CDA, we always support both HTTP and HTTPS.